### PR TITLE
Support for connection in model generator

### PR DESCRIPTION
**Pull request by zambodaniel**
refs. #168

### CODE REVIEW

File: templates/model/model.stub [hunk 1]
```diff
@@ -8,6 +8,7 @@ $DOCS$
 class $MODEL_NAME$ extends Model
 {
 $SOFT_DELETE$
+    public $connection = '$CONNECTION_NAME$';
```
Comment (mitulgolakiya) [inline templates/model/model.stub]: somehow, can we make it optional? As lots of people will not need it.

Comment (zambodaniel) [reply templates/model/model.stub]: Sure, i'll update this soon.

File: src/Utils/TableFieldsGenerator.php [hunk 2]
```diff
@@ -32,6 +32,8 @@ class TableFieldsGenerator
     public $tableName;
     public $primaryKey;
 
+    public static $connectionName;
```
[the remark this answers is not in the document]
Comment (zambodaniel) [reply src/Utils/TableFieldsGenerator.php]: @mitulgolakiya because getPrimaryKeyOfTable() function is static and needs a schemamanager.

**Comment by vishalinfyom** [comment]
@zambodaniel thanks for your contribution, as this PR has many conflict files we have merged this https://github.com/InfyOmLabs/laravel-generator/pull/762

**Pull request closed without merging**